packages/cli/cli/versions.yml

```diff
@@ -1,4 +1,12 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
+- changelogEntry:
+    - summary: |
+        Fix ts local github generation to match remote generation.
```

Comment:
=> `Fix TypeScript SDK generator's local GitHub generation to match remote generation.`